packages/core/src/Stores/client-store.js

```diff
@@ -615,6 +618,129 @@ export default class ClientStore extends BaseStore {
+                dark: 'IcWalletCurrencyGbp',
+                light: 'IcWalletCurrencyGbp',
+            },
+            BTC: {
+                dark: 'IcWalletBitcoinDark',
+                light: 'IcWalletBitcoinLight',
+            },
+            ETH: {
+                dark: 'IcWalletEthereumDark',
+                light: 'IcWalletEthereumLight',
+            },
+            USDT: {
+                dark: 'IcWalletTetherDark',
+                light: 'IcWalletTetherLight',
+            },
+            eUSDT: {
+                dark: 'IcWalletTetherDark',
+                light: 'IcWalletTetherLight',
+            },
+            tUSDT: {
+                dark: 'IcWalletTetherDark',
+                light: 'IcWalletTetherLight',
+            },
+            UST: {
+                dark: 'IcWalletTetherDark',
+                light: 'IcWalletTetherLight',
+            },
+            LTC: {
+                dark: 'IcWalletLiteCoinDark',
+                light: 'IcWalletLiteCoinLight',
+            },
+            USDC: {
+                dark: 'IcWalletUsdCoinDark',
+                light: 'IcWalletUsdCoinLight',
+            },
+        };
+
+        return this.all_loginids
+            .filter(id => this.getAccount(id)?.account_category === 'wallet')
+            .map(id => {
+                const account = this.getAccount(id);
+                const loginid = id;
+                const currency = account.currency;
+                const is_disabled = Boolean(account.is_disabled);
+                const is_virtual = Boolean(account.is_virtual);
+
+                const fiat_or_crypto = this.is_crypto(currency) ? 'crypto' : 'fiat';
+                const icon_type = is_virtual ? 'demo' : fiat_or_crypto;
+                const landing_company_name = account.landing_company_name?.replace('maltainvest', 'malta');
+                const is_malta_wallet = landing_company_name === 'malta';
+                const dtrade_loginid = account?.linked_to?.find(account => account?.platform === 'dtrade')?.loginid;
+                const dtrade_balance = this.accounts?.[dtrade_loginid]?.balance;
+
+                const wallet_currency_type = is_virtual ? 'Demo' : currency || '';
+                const icons = currency_to_icon_mapper[wallet_currency_type];
+
+                const gradients = {
+                    /** The gradient class name for the wallet header background. */
+                    header: {
+                        dark: `wallet-header__${wallet_currency_type.toLowerCase()}-bg--dark`,
+                        light: `wallet-header__${wallet_currency_type.toLowerCase()}-bg`,
+                    },
+                    /** The gradient class name for the wallet card background. */
+                    card: {
+                        dark: `wallet-card__${wallet_currency_type.toLowerCase()}-bg--dark`,
+                        light: `wallet-card__${wallet_currency_type.toLowerCase()}-bg`,
+                    },
+                };
```

Comment:
Is this necessary to put classnames and icons in the store?

Reply:
Sergei has moved this code in the hooks in his PR